### PR TITLE
Minor README updates

### PR DESCRIPTION
**Pull request by jarrpa**
Follow-up to #20 and #21.

Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

**Comment by jarrpa** [comment]
@msau42 @saad-ali PTAL

File: README.md [hunk 1]
```diff
@@ -9,49 +9,38 @@ for the CSI driver.
...
-
-  * The cluster-driver-registrar can indicate that the associated driver wants
-    kubelet to send additional information about the driver Pod as
-    VolumeAttributes on a volume mount.
-  <!-- TODO: Upload pod info details to csi-docs, reference here -->
-  * This is controlled by the `--pod-info-mount-version` argument. The
-    supported values are:
-    * `v1`
-
 ### Required arguments
```
Comment (msau42) [inline README.md]: Can you change this title to "Common arguments"?

File: README.md [hunk 2]
```diff
@@ -9,49 +9,38 @@ for the CSI driver.
...
 
-Though not strictly required, the following parameter it typically customized:
+Though not strictly required, the following parameters are typically
+customized:
 
 * `--csi-address`: This is the path to the CSI driver UNIX domain socket inside
   the pod that the `cluster-driver-registrar` container will use to issue CSI
   operations (e.g. `/csi/csi.sock`).
+* `--pod-info-mount-version`: This controls what Pod information is passed to
```
Comment (msau42) [inline README.md:41]: Add another sentence here saying that this should only be set if you require Kubernetes pod information.

**Comment by jarrpa** [comment]
@msau42 Force pushed with the requested changes.

**Pull request merged**